app/presenters/tree_builder_alert_profile_obj.rb

```diff
@@ -63,18 +59,14 @@ def root_options
   end
 
   def x_get_tree_roots(count_only, _options)
-    @objects = []
-    if !(@assign[:new][:assign_to] == "enterprise") && @assign[:new][:assign_to]
-      # No further selection needed for enterprise
-      # Assign to selected
-      if @assign[:new][:assign_to].ends_with?("-tags") && @assign[:new][:cat]
-        # Tag category selected
-        @objects = Classification.find(@assign[:new][:cat]).entries
-      else
-        # Model selected
-        @objects = @assign[:new][:assign_to].camelize.constantize.all
-      end
-    end
+    @objects = if !@assign[:new][:assign_to] || @assign[:new][:assign_to] == "enterprise"
```

Comment:
You are right, i must get used to the fact, that those with @ are not all variables but only instance variables...